Condition 13 method #14

paulinegarelli opened the pull request:
Should only be merged once Condition 8 (#10) (branch pauline/feat/10-condition8) has been merged.

Commit:
feat: Condition 13 method #14

hampfh commented on decide/src/main/java/org/example/Decide.java:
```diff
@@ -111,7 +111,12 @@ public boolean condition12() {
+                (index) -> Point.smallestCircleRadius(settings.POINTS[index],
+                        settings.POINTS[index + settings.PARAMETERS.A_PTS + 1],
+                        settings.POINTS[index + settings.PARAMETERS.A_PTS + settings.PARAMETERS.B_PTS + 2])
+                        < settings.PARAMETERS.RADIUS2);
```
I think it should be `<= settings.PARAMETERS.RADIUS2` since we allow points to be on the line too

Commit:
fix: non-strict comparison to radius 2 #14

In condition 13, the points are considered to be contained in a circle even if they are on the line, so the comparison should be not strict.

paulinegarelli committed:
Merge branch 'main' into pauline/feat/14-condition13

hampfh reviewed (approved):
👍
